### PR TITLE
DDP-4141 osteo: add validations on DOB

### PR DESCRIPTION
**Pull request by yufengwng**
## Context

* Added activity validations on each consent's DOB so we ensure age is good for that workflow
* Added a custom-task that we can run in production

## Checklist

- [x] I have labeled the type of changes involved using the `C-*` labels.
- [x] I have assessed potential risks and labeled using the `R-*` labels.
- [x] I have considered error handling and alerts, and added `L-*` labels as needed.
- [x] I have considered security and privacy, and added `I-*` labels as needed
- [x] I have analyzed my changes for stability, fault tolerance, graceful degradation, performance bottlenecks and written a brief summary in this PR.
- [x] (n/a) If applicable, I have discussed the analytics needs at both a platform and study level with Product and instrumented code accordingly.
- [x] (n/a) If applicable, my UI/UX changes have passed muster with Product/Design via an over-the-shoulder review, screenshots, etc.

_If unsure or need help with any of the above items, add the `help wanted` label. For items that starts with `If applicable`, if it is not applicable, check it off and add `n/a` in front._

## FUD Score

_Overall, how are you feeling about these changes?_

- [x] :relaxed: All good, business as usual!
- [ ] :sweat_smile: There might be some issues here
- [ ] :scream: I'm sweaty and nervous

## How do we demo these changes?

_How does one observe these changes in a deployed system? Note that **user visible** encompasses many personas--not just patients and study staff, but also ops duty, your fellow devs, compliance, etc._

- [ ] They are user-visible in dev as a regular user journey and require no additional instructions.
- [x] Getting dev into a state where this is user-visible requires some tech fiddling. I will document steps in release notes.
- [ ] Requires other features before it's human visible. I have documented the blocking issues in jira.
- [ ] I have no idea how to demo this. Please help me!

## Testing

- [ ] I have written automated positive tests
- [ ] I have written automated negative tests
- [x] I have written zero automated tests but have poked around locally to verify proper functionality
- [ ] The jira ticket has acceptance criteria and QA has the needed information to test changes

## Release

- [ ] These changes require no special release procedures--just code!
- [x] Releasing these changes requires special handling and I have documented the special procedures in the release plan document

### CODE REVIEW

File: study-builder/studies/osteo/patches/dob-validations.conf [hunk 1]
```diff
@@ -0,0 +1,149 @@
...
+          "messageTemplate": {
+            "templateType": "HTML",
+            "templateText": "$osteo_consent_dob_validation",
+            "variables": [
+              {
+                "name": "osteo_consent_dob_validation",
+                "translations": [
+                  {
+                    "language": "en",
+                    "text": "(TBD) You're not old enough for self consent!"
```
Comment (yufengwng) [inline study-builder/studies/osteo/patches/dob-validations.conf]: Still waiting on actual message to use, and how many validations to add, but the code is ready for review.

**Comment by yufengwng** [comment]
please run tests

**Comment by yufengwng** [comment]
please run tests

**Review by ssettipalli — APPROVED** [review]
looks good to me

**Pull request merged**